src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/CreateFileFromTemplateServiceTests.cs

```diff
@@ -59,11 +59,10 @@ public async Task CreateFile_NullFileName_ThrowsArgumentNull()
 
 
         [Theory]
-        [InlineData(@"Root (flags: {ProjectRoot}), FilePath: ""C:\Foo\""", @"C:\Path\To\TemplateFile", true)]
-        [InlineData(@"Properties, FilePath: ""C:\Foo\""", @"C:\Path\To\TemplateFile", true)]
-        [InlineData(@"Root (flags: {ProjectRoot}), FilePath: ""C:\Foo\""", @"C:\Path\To\TemplateFile", false)]
-        [InlineData(@"Root (flags: {ProjectRoot}), FilePath: ""C:\Foo\""", null, false)]
-        public async Task CreateFile(string input, string templateFilePath, bool expectedResult)
+        [InlineData(@"C:\Path\To\TemplateFile", true)]
+        [InlineData(@"C:\Path\To\TemplateFile", false)]
+        [InlineData(null, false)]
+        public async Task CreateFile(string templateFilePath, bool expectedResult)
```

Comment:
`input` was unused after [this change](https://github.com/dotnet/project-system/commit/425106035bb404a5ef720e664ad073c2dedf5324) that removed the project tree parsing, but it does make this test a little smelly IMO.. It seems to me that its basically asserting some of its own setup code. Also one of the sets of parameters was redundant.